config/defaults.conf

```diff
@@ -233,18 +233,17 @@ onlstm_tying = 0  // Language Modeling tying of weights.
 force_include_wsj_vocabulary = 0  // Set if using for PTB parsing task. Makes sure to include WSJ vocabulary.
 
 // Sentence Encoder //
-sent_enc = rnn  // Type of sentence encoder: 'bow', 'rnn' (for LSTM), or 'transformer'
-                // set to 'none' to do nothing but forward the skip connection.
+sent_enc = rnn  // Type of sentence encoder: 'bow', 'rnn' (for LSTM),, or 'transformer'
+                // set to 'none' to feed the output of the pretrained encoder into the task specific module.
                 // Note: Transformer has no known issues, but hasn't performed well, so there may be bugs.
-                // Note: 'bow' just skips the encoder step and passes the word representations to the task model, s
+                // Note: 'bow' just skips the encoder step and passes the word representations to the task model,
                 //   so it is possible to combine attention or max pooling with the 'bow' encoder.
 bidirectional = 1  // If true, the RNN encoder should be bidirectional.
 d_hid = 1024  // Hidden dimension size (usually num_heads * d_proj for transformer)
 n_layers_enc = 2  // Number of encoder layers. Usually 8–12 for Transformer.
 skip_embs = 1  // If true, concatenate the encoder's input (ELMo or embeddings) with the encoder's output.
-sep_embs_for_skip = 0  // Whether the skip embedding uses the same embedder object as the original embedding (before skip).
-                       // Only makes a difference if we are using ELMo weights, where it allows the four tuned ELMo scalars
-                       // to vary separately for each target task.
+// Not supported for BERT and OpenAI GPT.
+elmo_skip_connection_per_task = 0  // Whether the skip embedding uses the same embedder object as the original embedding (before skip), used only for ELMO
```

Comment:
A little more descriptive: "Whether to use the pretraining layer-mixing weights in the skip connection for target tasks or target-task specific layer mixing weights.